src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,56 @@
+.stopwatch {
+  position: relative;
+  width: $stopwatch-size;
+  height: $stopwatch-size;
+
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    bottom: 50%;
+    left: center-stopwatch-elements($minutes-hand-width);
```

Comment:
The function `center-stopwatch-elements` should not be used with the `left` property. Instead, use `transform: translateX(-50%)` in combination with `left: 50%` to center the element horizontally.